draft-ietf-httpbis-safe-method-w-body.xml

```diff
@@ -218,8 +219,8 @@ q=foo&amp;limit=10&amp;sort=-published
       representation of the resource identified by the target URI
       (as defined by <xref target="HTTP" section="7.1"/>), the QUERY
       method is used to ask the server to perform a query operation
-      (described by the request content) over some set of data scoped to the
-      target URI. The content returned in response to a QUERY
+      (described by the request content) over some set of data at the
+      resource. The content returned in response to a QUERY
```

Comment:
That's the only change ("scoped" vs "at") where I'm not completely sure it's an improvement. For "scoped", did we intend to have this an hierarchical quality?

Reply:
I have no idea what this is supposed to imply with "scoped", honestly.  My conception of this feature is that you query a resource and it uses it state ("data at the resource") to answer the question.  The URI only identifies the resource.

Reply:
yes, "URI -> resource" is obvious.